src/main/java/org/reaktivity/command/log/internal/LogCountersCommand.java

```diff
@@ -84,17 +87,26 @@ private void counters(
         }
     }
 
-    void invoke()
+    void invoke() throws InterruptedException
     {
-        try (Stream<Path> files = Files.walk(directory, 2))
+        boolean hasInterval = true;
+        while (hasInterval)
         {
-            files.filter(this::isControlFile)
-                 .peek(this::onDiscovered)
-                 .forEach(this::counters);
-        }
-        catch (IOException ex)
-        {
-            LangUtil.rethrowUnchecked(ex);
+            if(interval <= 0)
+            {
+                hasInterval = false;
+            }
+            try (Stream<Path> files = Files.walk(directory, 2))
+            {
+                files.filter(this::isControlFile)
+                        .peek(this::onDiscovered)
+                        .forEach(this::counters);
```

Comment:
The ControlLayout call requires an `nmap` OS command to be made, which can greatly impact performance.   This implementation requires remapping of discovered files at every interval, taking the performance hit consistently.

Reply:
Perhaps cache the already discovered files and their mappings, so you don't remap